elide-core/src/main/java/com/yahoo/elide/core/HttpStatus.java

```diff
@@ -18,4 +18,5 @@
     public static final int SC_NOT_FOUND = 404;
     public static final int SC_LOCKED = 423;
     public static final int SC_INTERNAL_SERVER_ERROR = 500;
+    public static int SC_UPDATED = SC_NO_CONTENT;
```

Comment:
I agree with the previous comment I read: not sure this is how we want to do this. This configuration shouldn't be globally modifiable at-will. We should encapsulate this nicely inside of Elide.